### PR TITLE
ATO-176: Enable AIS actions flag in staging and add required env vars

### PR DESCRIPTION
**Pull request by billrdunn**
## What?

Enable the AIS actions flag in staging. This enables actions such as redirecting and auditing after a AIS response is received. 

Add the environment variables which are required when the flag is turned on.

## Why?

For testing in staging env.

### CODE REVIEW

**Review by isaac-GDS — DISMISSED** [review]

**Comment by isaac-GDS** [comment]
Have we fully tested the previous AIS flag ?

**Review by ethanmills — APPROVED** [review]
Are these still pointing at the orch frontend? When do we plan to change them?

**Comment by billrdunn** [comment]
> Are these still pointing at the orch frontend? When do we plan to change them?

Good point, the plan was to redirect to auth frontend until orch frontend is deployed in staging

**Comment by billrdunn** [comment]
This PR won't be merged until https://github.com/govuk-one-login/authentication-api/pull/3820 is merged.

**Review by ethanmills — APPROVED** [review]

File: ci/terraform/oidc/authentication-callback.tf [hunk 1]
```diff
@@ -48,6 +48,8 @@ module "authentication_callback" {
     REDIS_KEY                                   = local.redis_key
     SUPPORT_AUTH_ORCH_SPLIT                     = var.support_auth_orch_split
     TXMA_AUDIT_QUEUE_URL                        = module.oidc_txma_audit.queue_url
+    OIDC_API_BASE_URL                           = local.api_base_url
```
Comment (isaac-GDS) [inline ci/terraform/oidc/authentication-callback.tf:51]: is this needed then ?

Comment (isaac-GDS) [reply ci/terraform/oidc/authentication-callback.tf:51]: needed for other envs

**Review by isaac-GDS — APPROVED** [review]

File: ci/terraform/oidc/authentication-callback.tf (hunk 1 above)
Comment (billrdunn) [reply ci/terraform/oidc/authentication-callback.tf:51]: yep it's the base redirect URL in envs other than staging and prod

**Pull request merged**